docs/hardware/devices/rak/core-modules.mdx

```diff
@@ -116,11 +116,11 @@ The RAK11200 does not contain a LoRa transceiver, and thus needs to be added sep
 - Firmware file: `firmware-rak11200-X.X.X.xxxxxxx.bin`
 - Further information on the RAK11200 can be found on the [RAK Documentation Center](https://docs.rakwireless.com/Product-Categories/WisBlock/RAK11200/Overview/#product-description).
 - Purchase Links:
+  - US
+      - [Rokland](https://store.rokland.com/products/rakwireless-rak11200-wifi-and-ble-espressif-esp32-wrover-pid-110023)
```

Comment:
Indentation doesn't match here, think you have an extra space or two in both of these files